R/gdef.R

```diff
@@ -27,6 +27,8 @@ gdef <- function(term, text){
     language <- glosario_data$language
   }
 
-  string <- glue::glue('<a href="{base_url}/{language}/#{term}" class="glossary-definition">{text}</a>')
+  # We introduce a div with class glosario_def so people can style HTML
+  # The link itself is a MD link so that it renders without problem in all formats.
+  string <- glue::glue('<div class="glosario_def">[{text}]({base_url}/{language}/#{term})</div>')
```

Comment:
Should this be a `span` instead of a `div` because it's inline?

Reply:
I changed it in the new commits. Cheers